.github/workflows/wheels.yml

```diff
@@ -138,45 +146,13 @@ jobs:
         with:
           name: requirements_diff
 
-      - name: (Un)comment packages
-        run: |
-          requirement_files="requirements_all.txt requirements_diff.txt"
-          for requirement_file in ${requirement_files}; do
-            sed -i "s|# fritzconnection|fritzconnection|g" ${requirement_file}
-            sed -i "s|# pyuserinput|pyuserinput|g" ${requirement_file}
-            sed -i "s|# evdev|evdev|g" ${requirement_file}
-            sed -i "s|# pycups|pycups|g" ${requirement_file}
-            sed -i "s|# homekit|homekit|g" ${requirement_file}
-            sed -i "s|# decora-wifi|decora-wifi|g" ${requirement_file}
-            sed -i "s|# python-gammu|python-gammu|g" ${requirement_file}
-
-            # Some packages are not buildable on armhf anymore
-            if [ "${{ matrix.arch }}" = "armhf" ]; then
-
-              # Pandas has issues building on armhf, it is expected they
-              # will drop the platform in the near future (they consider it
-              # "flimsy" on 386). The following packages depend on pandas,
-              # so we comment them out.
-              sed -i "s|env-canada|# env-canada|g" ${requirement_file}
-              sed -i "s|noaa-coops|# noaa-coops|g" ${requirement_file}
-              sed -i "s|pyezviz|# pyezviz|g" ${requirement_file}
-              sed -i "s|pykrakenapi|# pykrakenapi|g" ${requirement_file}
-            fi
-
-            # Some speedups are only for 64-bit
-            if [ "${{ matrix.arch }}" = "amd64" ] || [ "${{ matrix.arch }}" = "aarch64" ]; then
-              sed -i "s|aiohttp-zlib-ng|aiohttp-zlib-ng\[isal\]|g" ${requirement_file}
-            fi
```

Comment:
I don't think this worked, because `aiohttp-zlib-ng` is included in `requirements.txt`, not in `requirements_all.txt`

Reply:
We can drop this and go directly with `aiohttp-zlib-ng[isal]` as it works on 32 bit now